modules/openapi-generator/src/main/resources/python-pydantic-v1/model_generic.mustache

```diff
@@ -10,6 +10,16 @@ import json
 {{{.}}}
 {{/vendorExtensions.x-py-model-imports}}
 
+{{#hasChildren}}{{#discriminator}}
```

Comment:
thanks for the PR.

minor suggestion: 

1) use the following to avoid the empty line break

{{#hasChildren}}
{{#discriminator}}
...
{{/discriminator}}
{{/hasChildren}}

2) for line 22 (empty line), move it between line 20 and 21 so that it will only be included when both hasChilden and discriminator are non-emtpy (or true)

Reply:
This issue has been fixed.